src/main/java/it/gov/pagopa/fdr/to/eventhub/mapper/FlussoRendicontazioneMapper.java

```diff
@@ -40,6 +38,9 @@ public class FlussoRendicontazioneMapper {
           .appendPattern("XXX")
           .optionalEnd()
           .toFormatter();
+  @Getter
+  @Setter
+  private static int maxDistinctDates = 110;
```

Comment:
why 110?

Reply:
I didn't set it in this PR, but IIRC this value was set in order to avoid the insertion of all payment's dates in single flow.